SwiftyJSON.podspec

```diff
@@ -11,6 +11,6 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = "8.0"
   s.watchos.deployment_target = "2.0"
   s.tvos.deployment_target = "9.0"
-  s.source   = { :git => "https://github.com/SwiftyJSON/SwiftyJSON.git", :tag => s.version }
```

Comment:
Honestly I don't remember why I did this. Maybe the master repo was more updated than the latest publish revision and so I needed that version of the code.
Probably right now, this change can be reverted.